packages/apps/graph/src/devnet/helper.ts

```diff
@@ -111,11 +111,20 @@ export const generateAccount = async (
   let account = `k:${keyPairs[0].publicKey}`;
 
   if (keyPairs.length > 1) {
-    account = await createPrincipal({
-      keys: keyPairs.map((keyPair) => keyPair.publicKey),
-      pred: 'keys-all',
-      chainId: chainId,
-    });
+    account = await createPrincipal(
+      {
+        keyset: {
+          keys: keyPairs.map((keyPair) => keyPair.publicKey),
+        },
+      },
+      {
+        host: `http://localhost:${devnetConfig.PORT}`,
+        defaults: {
+          networkId: devnetConfig.NETWORK_ID,
+          meta: { chainId },
+        },
+      },
+    );
```

Comment:
Can we make it explicit instead of relying on overwriting an existing value?

E.g.: 

```
  let account: string | undefined = undefined;
  
  if (keyPairs.length === 1) {
    account = `k:${keyPairs[0].publicKey}`;
  } else if (keyPairs.length > 1) {
    account = await createPrincipal(
      {
        keyset: {
          keys: keyPairs.map((keyPair) => keyPair.publicKey),
        },
      },
      {
        host: `http://localhost:${devnetConfig.PORT}`,
        defaults: {
          networkId: devnetConfig.NETWORK_ID,
          meta: { chainId },
        },
      },
    );
```